src/styles/card.scss

```diff
@@ -0,0 +1,113 @@
+@import "./_variables.scss";
+
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  margin: 0;
+  margin-left: 48px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:first-child {
+    margin-left: 0;
+  }
+}
+
+.card__imac {
```

Comment:
Make use of the sass nesting, this should be `&__imac`, fix it everywhere

Reply:
I.. honestly did not know it could be used for object children as well, every example just used it with functions like :hover, so I assumed that's what that's for
My bad, I will make use of it properly from now on